package.json

```diff
@@ -24,9 +24,5 @@
   "homepage": "https://github.com/TBDG/primal-shop#readme",
   "devDependencies": {
     "concurrently": "^5.3.0"
-  },
-  "dependencies": {
-    "dotenv": "^8.2.0",
```

Comment:
the dotenv package needs to stay in the root package.json only the react-loader-spinner needs to be moved